ckanext/datajson/tests/mock_datajson_source.py

```diff
@@ -1,18 +1,18 @@
 from __future__ import print_function
 
-import os
 import json
-import re
-import copy
-import urllib
-
+import logging
+import os
 import SimpleHTTPServer
 import SocketServer
 from threading import Thread
-import logging
+
+from ckanext import datajson
+
 log = logging.getLogger("harvester")
 
 PORT = 8998
+SAMPLES_PATH = os.path.join(datajson.__path__[0], 'tests/datajson-samples')
```

Comment:
Same comment as https://github.com/GSA/ckanext-geodatagov/pull/123#discussion_r375941117, there should be a better way to do this so it doesn't break in all these corner cases.

Reply:
I can't use the file method here because datajson is being declared as a namespace package in the init file